validator/rpc/server.go

```diff
@@ -132,6 +136,20 @@ func NewServer(ctx context.Context, cfg *Config) *Server {
 		validatorGatewayPort:     cfg.ValidatorGatewayPort,
 		router:                   cfg.Router,
 	}
+
+	if server.authTokenPath == "" && server.walletDir != "" {
+		server.authTokenPath = filepath.Join(server.walletDir, api.AuthTokenFileName)
+	}
+
+	if server.authTokenPath != "" {
+		log.Infof("at server start %s", server.authTokenPath)
```

Comment:
```
[2024-03-18 11:07:50]  INFO rpc: at server start /Users/manu/Library/Eth2Validators/prysm-wallet-v2/auth-token
```

This log does not start with a capital letter and is not very clear to me.

Reply:
this should be removed*